content/tests/components/recipes/details/RecipeDetails.test.js

```diff
@@ -166,6 +186,14 @@ describe("The `RecipeDetails` component", () => {
       ).not.toHaveLength(0);
       expect(doc.getAllByText(branch.ratio.toString())).not.toHaveLength(0);
     }
+
+    for (const extension of extensions) {
+      const nameRegex = new RegExp(extension.name);
+      const versionRegex = new RegExp(extension.version);
+
+      expect(doc.getByText(nameRegex)).toBeInTheDocument();
+      expect(doc.getByText(versionRegex)).toBeInTheDocument();
```

Comment:
Why use regex here, instead of just using the name and version directly in the `getByText` call?

Reply:
i mushed the name and version together so that it's `<name> (version : <version>)` and `getByText` needs a regex for substrings?

Reply:
Ah. That will work, but there is a simpler way: `expect(doc.getByText(extension.name, {exact: false})`. That will let it match partial elements.